### PR TITLE
charts: Add schema for values.yaml

### PR DESCRIPTION
**Pull request by knrt10**
Charts did not have defined structure for Headlamp value configurations. Now we have defined schema for the same which will run while linting, installing, upgrading or templating the helm charts.

Fixes: #1926

### CODE REVIEW

File: charts/headlamp/values.yaml [hunk 1]
```diff
@@ -84,7 +84,7 @@ config:
   # -- directory to look for plugins
   pluginsDir: "/headlamp/plugins"
   # Extra arguments that can be given to the container. See charts/headlamp/README.md for more information.
-  extraArgs: {}
+  extraArgs: []
```
Comment (joaquimrocha) [inline charts/headlamp/values.yaml:87]: This doesn't seem related to the patch.

**Review by joaquimrocha — CHANGES_REQUESTED** [review]

File: charts/headlamp/values.yaml (hunk 1 above)
Comment (knrt10) [reply charts/headlamp/values.yaml:87]: Fixed in a seperate commit. extraArgs should indeed be an array rather than an object. Since array can also contain object values.

**Review by joaquimrocha — APPROVED** [review]
